CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Nix
+
+A big welcome and thank you for considering contributing to Nix! It's people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+It also communicates that you agree to respect the time of the developers managing and developing these open source projects.
+In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Code of conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication.
+By participating and contributing to this project, you agree to uphold our [Code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/).
```

Comment:
We have [an initial CoC in the making](https://github.com/NixOS/rfcs/pull/114), but it's not merged yet. I wouldn't block this on the CoC, so let's remove it for now. We can always add it when it's there.

```suggestion
```